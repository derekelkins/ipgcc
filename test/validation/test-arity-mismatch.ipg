S -> 
    A(1, 2)
    for i = 0 to 3 do B[i, i+1]
    repeat C.y
    repeat D.y until E;

B(x) -> "a";
C(x) -> { y = "a" };
D(x) -> { y = "a" };
E(x) -> "a";

A(x) -> { y = . } ?[ x == y ];
