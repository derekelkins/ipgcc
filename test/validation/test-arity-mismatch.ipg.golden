Arity mismatch when calling A in rule S
Arity mismatch when calling B in rule S
Arity mismatch when calling C in rule S
Arity mismatch when calling D in rule S
Arity mismatch when calling E in rule S